src/features/envVisualizer/EnvVisualizerUtils.ts

```diff
@@ -425,7 +442,10 @@ export function getControlItemComponent(
           topItem
         );
       default:
+        console.log("Im not a program");
+        console.log("Control item: ", controlItem);
         const text = astToString(controlItem).trim();
+        console.log("text: ", text);
```

Comment:
Could we remove the console.log statements?

Reply:
Oops I forgot to remove these, I will format the files again as well